index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" http-equiv="refresh" content="2; URL=http://arquillian.org/arquillian-cube/#_kubernetes">
```

Comment:
Can you change to 0? So users only see the message only if there is some problem on browser.